resources/wif/4.17/vanilla.yaml

```diff
@@ -302,6 +302,9 @@ service_accounts:
           - storage.objects.delete
           - storage.objects.get
           - storage.objects.list
+          - resourcemanager.tagValueBindings.create
+          - resourcemanager.tagValues.get
+          - resourcemanager.tagValues.list
```

Comment:
These are already included a few lines above. :) 

FYI: We created this WIF config as part of the WIF epic, the usual expectation when following the Gap Analysis SOP would be that this file doesn't yet exist - so this is a special case.